README.md

```diff
@@ -95,13 +118,9 @@ As you can see, you can mix and match dsc resources with common puppet resources
 All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metaparameter.html) should also be supported.
 
 ## Limitations
-- DSC Composite resources not yet fully supported.
+- DSC Composite Resources are not yet fully supported.
 - PS Objects like 'PSCredential' as parameters value not yet supported.
 
 ## Notes
-The puppet types are build from the source code of the DSC Resources MOF schema files.
-If you want the build Puppet types for your own custom DSC Resources, read the [README_BUILD](https://github.com/msutter/puppet-dsc/blob/master/README_BUILD.md)
-
-## License
```

Comment:
I would suggest an attributions section -  https://github.com/chocolatey/puppet-chocolatey/blob/master/README.md#attributions  

For copyright, I think it gets weird. It's kind of a copyright 2014 (c) Marc Sutter, copyright 2014-2015 (c) Puppet Labs. Does this seem correct @stahnma or am I completely off?
In a multiple copyright situation, we may need to put copyrights on every file.